src/components/donation/Steps/Submit/ChariotConnectCheckout.tsx

```diff
@@ -33,8 +36,13 @@ export default function ChariotConnectCheckout(props: ChariotCheckoutStep) {
     };
   };
 
+  const currency = details.currency;
+  const total = +details.amount;
+  const liq = total * (liquidSplitPct / 100);
+  const locked = total - liq;
+
   return (
-    <div className="grid content-start gap-8 p-4 @md:p-8">
+    <div className="flex flex-col content-start p-4 @md:p-8 group">
       <BackBtn
```

Comment:
Bug: when the user clicks "Back", they should be redirected to the Split component (as is the case with all other donation methods); instead, the user is navigated all the way back to currency+donation amount screen.

Reply:
fixed: https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2709/commits/c87b9b48868d61e31e5d00199520180f907ecff0